tests/unit/amo/components/TestRatingManager.js

```diff
@@ -124,6 +124,38 @@ describe(__filename, () => {
     sinon.assert.notCalled(loadSavedReview);
   });
 
+  it('renders loading stars before the saved review has loaded', () => {
+    const addon = createInternalAddon({ ...fakeAddon });
+    const { store } = dispatchSignInActions();
+
+    const root = render({ addon, store });
+
+    expect(root.find(UserRating)).toHaveProp('review', undefined);
```

Comment:
I see what you mean about the impact of your recent change. The code does seem cleaner, but it's a bit less clear exactly how it's working. For example, in this test, it's described as "renders loading stars..." but doesn't have an assertion that seems to match (i.e., the assertion doesn't say anything about loading, rather it's about the contents of the `review` prop).

Maybe the descriptions of these tests should be updated to describe what we actually expect, which is a specific value for the `review` prop?

Reply:
Yeah, I can see how changing these descriptions could be helpful. The loading state is now the responsibility of `Rating`.